src/components/nve-select/nve-select.component.ts

```diff
@@ -88,9 +88,7 @@ export default class NveSelect extends SlSelect {
 
       if (this.value !== oldValue) {
         this.updateComplete.then(() => {
-          // @ts-expect-error - this is shadowed by outer container
```

Comment:
Disse to linjene er ikke direkte relatert til denne saken. Men tsc nektet å kompilere med disse